common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -3919,49 +3924,56 @@ schematic = (function() {
 	    var tool = this.tools[tname];
 
 	    if (tool != undefined) {
-		tool.style.opacity = which ? 1.0 : 0.2;
+        tool.removeAttribute('disabled');
 		tool.enabled = which;
 
 		// if disabling tool, remove border and tip
 		if (!which) {
-		    tool.style.borderColor = background_style;
+		    // tool.style.borderColor = background_style;
```

Comment:
This line should just be removed completely rather than commented out if it's not needed anymore.